app/konnect/dedicated-cloud-gateways.md

```diff
@@ -0,0 +1,63 @@
+| Your organization needs high availability with zero downtime while upgrading data plane nodes. | {{site.konnect_short_name}} handles upgrades for you. There's no downtime when upgrading the infrastructure because when Kong performs the rolling upgrade, we synchronize the data plane node with load balancer registration and deregistration and gracefully terminate the old data plane nodes to reduce the impact on ongoing traffic. Additionally, you can pre-warm your cluster by specifying the number of requests per second so that the first requests don’t have to wait for the infrastructure to scale up. |
+| Your organization uses several tools for API gateways and their infrastructure and you  want to reduce operational complexity. | Kong manages the cluster for you, all you need to do is configure the {{site.base_gateway}} version, configuration mode, cluster region, and the API access level. |
+| You have infrastructure in multiple clouds. | Dedicated Cloud Gateways allows you to run a multi-cloud solution that allows you to standardize API operations across the board to reduce complexity and increase agility. |
+
+## How do Dedicated Cloud Gateways work?
+
+{% include_cached /md/konnect/deployment-topologies.md topology='cloud' %}
+
+> _**Figure 2:** In a dedicated cloud gateway, the control plane is hosted in {{site.konnect_short_name}} and data planes are hosted in the cloud by Kong. The control plane connects to the database, and the data planes receive configuration from the control plane._
+
+When you create a Dedicated Cloud Gateway, {{site.konnect_short_name}} also creates a control plane for the cloud gateway. This control plane, like other {{site.konnect_short_name}} control planes, is hosted in {{site.konnect_short_name}} using the AWS infrastructure. Kong also creates, and manages, the data plane nodes in the AWS region that you specify while configuring the Dedicated Cloud Gateway.
+
+When you configure your Dedicated Cloud Gateway, you can choose one of two configuration modes to create your data plane nodes:
+
+* **Autopilot:** In Autopilot mode, you configure how many requests per second you expect the instance to recieve, then Kong pre-warms and autoscales the data plane nodes in the cluster for you.
+* **Custom:** In Custom mode, you specify the instance size and type (for example, dev, production, or large production) as well as the number of nodes per cluster.
+
+Because data plane nodes in Autopilot configuration mode are auto scaled, you **cannot** manually increase or decrease nodes. You can only manually increase or decrease data plane nodes when the Dedicated Cloud Gateway is configured in the Custom mode.
```

Comment:
```suggestion
Because data plane nodes in Autopilot configuration mode automatically scale, you **cannot** manually increase or decrease nodes. You can only manually increase or decrease data plane nodes when the Dedicated Cloud Gateway is configured in the Custom mode.
```